### ISSUE TITLE
Add an indication of toots still not seen

### ISSUE THREAD

**Issue by javerous** [metabolist/metatext#197]
**Is your feature request related to a problem? Please describe.**
Not really.

**Describe the solution you'd like**
It would be nice to see the number of toots which wasn't seen / read, a bit like an unread count badge in an e-mail client. I'm not sure it's doable with the current protocol.

**Describe alternatives you've considered**
An alternative, if it's not possible currently, would be to show the number of toots between the current timeline position, and last toot of the timeline. Not sure it's possible neither with the current protocol.

**Additional context**
It's something which is already implemented in some Twitter clients, like Tweetbot. They show it as an overlay (top right) on the timeline, which is a good solution, I think.

<img width="342" alt="Screenshot 2022-11-18 at 16 42 09" src="https://user-images.githubusercontent.com/1971659/202744003-cef8640b-8862-4fa3-8a92-4f7a019790b5.png">

**Comment by fe-hicking** [comment]
Just came here to see if that feature request had been made already. I'm a 100% timeline reader too, and would benefit from it. Especially if it could show total unread state. Currently I have to "load more" my way above the fold. Would also be great if background fetching would "load more" on its own, so that I could do continuous scrolling.

Sadly am only a PHP developer and know nothing about Xcode development, so hard for me to offer help, if I can't even build a local version ;-)

**Comment by javerous** [comment]
> Would also be great if background fetching would "load more" on its own, so that I could do continuous scrolling.

Yes, I was about to open a feature request for that. It can be something like: if you slowly approaching the "load more zone", then automatically load in the background (because this means you are reading toots, and you will want to read the ones missing, so it makes sense to preload them). And if you are scrolling fast, then don't load them (you are probably just skipping your timeline and trying to reach new / old toots, not reading, so it makes sense to skip the ones missing).


> Sadly am only a PHP developer and know nothing about Xcode development, so hard for me to offer help, if I can't even build a local version ;-)

I'm a macOS / iOS developer myself, but I don't have much free time. If nobody from the core team have time to do it (or are not interested to do it), I will see if I can implement these 2 features at some point.

**Comment by mykter** [comment]
![image](https://user-images.githubusercontent.com/1424497/203542772-2899e116-d5f6-427e-963d-91d5ef2af66e.jpeg)
This is how Twitterrific displays the unread count - and in their case  it means not seen, not just distance to latest post.
